packages/w3/index.js

```diff
@@ -1,8 +1,71 @@
 import { globSource } from './utils/glob-source.js'
 import { writeFiles } from 'ipfs-car/unpack/fs'
 import { Web3Storage } from 'web3.storage'
+import enquirer from 'enquirer'
+import Conf from 'conf'
 import ora from 'ora'
 
+const API = 'https://api.web3.storage'
+const config = new Conf()
+
+/**
+ * Get a new API client configured either from opts or config
+ * @param {object} opts
+ * @param {string} [opts.api]
+ * @param {string} [opts.token]
+ */
+function getClient ({
+  api = config.get('api') || API,
+  token = config.get('token')
```

Comment:
I would show the prompt if not existent and drop support to add via CLI param

Reply:
hmm maybe that is a good compromise tho. catches the cases where you didn't know you had to set a token too,